skema/skema-rs/mathml/tests/75.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" ?>
```

Comment:
Can you add a unit test for this file in `lib.rs`?

Reply:
And perhaps rename the file to something more descriptive than `75.xml`? Even something like `test_1.xml` would be fine.